backend/.env.dist

```
@@ -1,57 +1,60 @@
-
-# Server
-PORT=4000
-JWT_SECRET=secret123
-JWT_EXPIRES_IN=10m
-GRAPHIQL=false
-GDT_API_URL=https://gdt.gradido.net
-
-# Database
-DB_HOST=localhost
-DB_PORT=3306
-DB_USER=root
-DB_PASSWORD=
-DB_DATABASE=gradido_community
-TYPEORM_LOGGING_RELATIVE_PATH=typeorm.backend.log
-
-# Klicktipp
-KLICKTIPP=false
-KLICKTTIPP_API_URL=https://api.klicktipp.com
-KLICKTIPP_USER=gradido_test
-KLICKTIPP_PASSWORD=secret321
-KLICKTIPP_APIKEY_DE=SomeFakeKeyDE
-KLICKTIPP_APIKEY_EN=SomeFakeKeyEN
-
-# Community
-COMMUNITY_NAME=Gradido Entwicklung
-COMMUNITY_URL=http://localhost/
-COMMUNITY_REGISTER_URL=http://localhost/register
-COMMUNITY_REDEEM_URL=http://localhost/redeem/{code}
-COMMUNITY_REDEEM_CONTRIBUTION_URL=http://localhost/redeem/CL-{code}
-COMMUNITY_DESCRIPTION=Die lokale Entwicklungsumgebung von Gradido.
-
-# Login Server
-LOGIN_APP_SECRET=21ffbbc616fe
-LOGIN_SERVER_KEY=a51ef8ac7ef1abf162fb7a65261acd7a
-
-# EMail
-EMAIL=false
-EMAIL_USERNAME=gradido_email
-EMAIL_SENDER=info@gradido.net
-EMAIL_PASSWORD=xxx
-EMAIL_SMTP_URL=gmail.com
-EMAIL_SMTP_PORT=587
-EMAIL_LINK_VERIFICATION=http://localhost/checkEmail/{optin}{code}
-EMAIL_LINK_SETPASSWORD=http://localhost/reset-password/{optin}
-EMAIL_LINK_FORGOTPASSWORD=http://localhost/forgot-password
-EMAIL_LINK_OVERVIEW=http://localhost/overview
-EMAIL_CODE_VALID_TIME=1440
-EMAIL_CODE_REQUEST_TIME=10
-
-# Webhook
-WEBHOOK_ELOPAGE_SECRET=secret
-
-# SET LOG LEVEL AS NEEDED IN YOUR .ENV
-# POSSIBLE VALUES: all | trace | debug | info | warn | error | fatal
-# LOG_LEVEL=info
+CONFIG_VERSION=v9.2022-07-07
+
+# Server
+PORT=4000
+JWT_SECRET=secret123
+JWT_EXPIRES_IN=10m
+GRAPHIQL=false
+GDT_API_URL=https://gdt.gradido.net
+
+# Database
+DB_HOST=localhost
+DB_PORT=3306
+DB_USER=root
+DB_PASSWORD=
+DB_DATABASE=gradido_community
+TYPEORM_LOGGING_RELATIVE_PATH=typeorm.backend.log
+
+# Klicktipp
+KLICKTIPP=false
+KLICKTTIPP_API_URL=https://api.klicktipp.com
+KLICKTIPP_USER=gradido_test
+KLICKTIPP_PASSWORD=secret321
+KLICKTIPP_APIKEY_DE=SomeFakeKeyDE
+KLICKTIPP_APIKEY_EN=SomeFakeKeyEN
+
+# Community
+COMMUNITY_NAME=Gradido Entwicklung
+COMMUNITY_URL=http://localhost/
+COMMUNITY_REGISTER_URL=http://localhost/register
+COMMUNITY_REDEEM_URL=http://localhost/redeem/{code}
+COMMUNITY_REDEEM_CONTRIBUTION_URL=http://localhost/redeem/CL-{code}
+COMMUNITY_DESCRIPTION=Die lokale Entwicklungsumgebung von Gradido.
+
+# Login Server
+LOGIN_APP_SECRET=21ffbbc616fe
+LOGIN_SERVER_KEY=a51ef8ac7ef1abf162fb7a65261acd7a
+
+# EMail
+EMAIL=false
+EMAIL_USERNAME=gradido_email
+EMAIL_SENDER=info@gradido.net
+EMAIL_PASSWORD=xxx
+EMAIL_SMTP_URL=gmail.com
+EMAIL_SMTP_PORT=587
+EMAIL_LINK_VERIFICATION=http://localhost/checkEmail/{optin}{code}
+EMAIL_LINK_SETPASSWORD=http://localhost/reset-password/{optin}
+EMAIL_LINK_FORGOTPASSWORD=http://localhost/forgot-password
+EMAIL_LINK_OVERVIEW=http://localhost/overview
+EMAIL_CODE_VALID_TIME=1440
+EMAIL_CODE_REQUEST_TIME=10
+
+# Webhook
+WEBHOOK_ELOPAGE_SECRET=secret
+
+# EventProtocol
+EVENT_PROTOCOL_ENABLED=true

```

Did we agree in turn this around?
```suggestion
EVENT_PROTOCOL_DISABLED=false
```